### PR TITLE
Kev/391 variable 

### PR DESCRIPTION
**Pull request by junhaow1**
# Pull Request Details

## What issue does this PR address

- CLEANUP: When a variable is selected set VARIABLE option #391

- Link to associated issue: #391 

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [x] Pre-exisiting lint errors noted: [HERE]
- [x] Integration test `make qtest.tmp` screenshot included in issue
- [ ] Tested on device:
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [ ] Added two reviewers

## Finalising

Once PR discussion is complete and reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Comment by junhaow1** [comment]
image

File: lib/features/cleanup/config.dart [hunk 1]
```diff
@@ -253,13 +250,12 @@ class CleanupConfigState extends ConsumerState<CleanupConfig> {
 
     return Column(
       children: [
+        // Space to the left of the configs.
```
Comment (gjwgit) [inline lib/features/cleanup/config.dart]: This is wrong? It is space to the top. The next space below is to the left? Why the change?

File: lib/features/cleanup/config.dart [hunk 2]
```diff
@@ -237,14 +237,11 @@ class CleanupConfigState extends ConsumerState<CleanupConfig> {
     // TODO yyx 20240807 what should we allow to be deleted?
 
     List<String> inputs = getInputsAndIgnoreTransformed(ref);
-
     String method = ref.read(cleanUpMethodProvider.notifier).state;
-
     // Retrieve the current selected variable and use that as the initial value
```
Comment (gjwgit) [inline lib/features/cleanup/config.dart:245]: These spaces in the code make it easier to read the code? Why remove them?

File: lib/features/cleanup/config.dart [hunk 3]
```diff
@@ -269,9 +265,7 @@ class CleanupConfigState extends ConsumerState<CleanupConfig> {
               },
               child: const Text('Delete from Dataset'),
             ),
-
```
Comment (gjwgit) [inline lib/features/cleanup/config.dart]: Similarly, the space makes the code more readable. Please don't remove them.

File: lib/utils/variable_chooser.dart [hunk 4]
```diff
@@ -32,24 +32,41 @@ Widget variableChooser(
...
     }).toList(),
-    // On selection as well as recording what was selected rebuild the
-    // visualisations.
+    // Use the enabled parameter to control the dropdown state.
+
+    enabled: enabled,
     onSelected: (String? value) {
-      // ref.read(selectedProvider.notifier).state = value ?? 'IMPOSSIBLE';
-      ref.read(stateProvider.notifier).state = value ?? 'IMPOSSIBLE';
-      // We don't buildAction() here since the variable choice might
```
Comment (gjwgit) [inline lib/utils/variable_chooser.dart:50]: Why was this comment removed?

**Review by gjwgit — APPROVED** [review]
